jobs.md

```diff
@@ -7,6 +7,6 @@ Come and work with us!
 Interested in working on open source and containers at a leading hyper-scale cloud? You can help build the future of Kubernetes at Microsoft! Stay tuned for future
 openings.
 
-## Microsoft Container Registry
-Want to work on a cloud service that delivers a billion container images a month? Help shape the future of container security software supply chain at Microsoft!
-Stay tuned for future openings.
+- [Senior Software Engineer](https://careers.microsoft.com/us/en/job/1501501)
```

Comment:
just curious, are you still hiring now at this moment?

Reply:
@andyzhangx For this position yes we are still hiring.